src/main/java/org/benetech/servicenet/ServiceNetApp.java

```diff
@@ -65,6 +71,14 @@ public void initApplication() {
             LOG.error("You have misconfigured your application! It should not " +
                 "run with both the 'dev' and 'cloud' profiles at the same time.");
         }
+
+        if (env.getProperty("login") != null && env.getProperty("orgs") != null) {
+            String userLogin = env.getProperty("login");
```

Comment:
if we don't provide user's login, let's distribute the organizations across random users